best-practices/native/nativescript/images.md

```diff
@@ -0,0 +1,72 @@
+To handle image in Nativescript sometimes looks hard and frustrating to do so.
+
+Doing some research I found this video from Alexander Ziskind:
+
+https://www.youtube.com/watch?v=KgK_d-1lG84&t=46s
```

Comment:
With links like this I usually put them in md format, so something like:

[Handling Images in Nativescript](https://www.youtube.com/watch?v=KgK_d-1lG84&t=46s)

Its more personal preference as it makes the document look a little cleaner

Reply:
I would agree, its more clear that its a link (both with the formatting and the clickable link) and it also gets picked-up by `awesome_bot` within the CircleCI process.

Reply:
Cool. It is fixed.